client/redux/destinationReducer.js

```diff
@@ -0,0 +1,27 @@
+
+import * as types from "./actionTypes.js";
+
```

Comment:
I'm gonna change the names for a few of the states in this file so that there's no conflict with the other states. Although that may not be necessary if we're understanding redux correctly, I'm going to do it just in case.